redaxo/src/addons/structure/pages/linkmap.php

```diff
@@ -79,6 +79,11 @@ function insertLink(link,name){
 $isRoot = $category_id === 0;
 $category = rex_category::get($category_id);
 
+// Hide minibar (REX 5.7+)
+if (class_exists('rex_minibar') && rex_minibar::getInstance()->isActive() === null) {
+    rex_minibar::getInstance()->setActive(false);
```

Comment:
brauchen wir hier auch den "current-page-is-popup-check"?

Reply:
Ich vermute nicht, weil die Linkmap konzeptionell ja immer nur ein Picker ist. Der Medienpool hingegen könnte Standalone sein.